src/Rule.php

```diff
@@ -1986,9 +1986,18 @@ public function cleanDBonPurge()
      * @param $last               is it the last rule ? (false by default)
      * @param $display_entities   display entities / make it read only display (false by default)
      * @param $active_condition   active condition used (default 0)
-     **/
-    public function showMinimalForm($target, $first = false, $last = false, $display_entities = false, $active_condition = 0)
-    {
+     * @param $display_criterias  display rule criterias (false by default)
+     * @param $display_actions    display rule actions(false by default)
+     **/
+    public function showMinimalForm(
+        $target,
+        $first = false,
+        $last = false,
+        $display_entities = false,
+        $active_condition = 0,
+        $display_criterias = 0,
+        $display_actions = 0
+    ) {
```

Comment:
Adding new parameter on a public method, even optional, is a bad idea on a bugfix version.

A change on this method will trigger a fatal error on `statecheck` plugin `Fatal error: Declaration of PluginStatecheckRule::showMinimalForm($target, $first = false, $last = false, $display_entities = false, $active_condition = 0) must be compatible with Rule::showMinimalForm($target, $first = false, $last = false, $display_entities = false, $active_condition = 0, $display_criterias = 0, $display_actions = 0)`.

A solution to handle this is to do something like this:
```suggestion
        $active_condition = 0
        // FIXME Uncomment this in GLPI 10.1
        // bool $display_criterias = false,
        // bool $display_actions = false
    ) {
    $display_criterias = func_get_args()[5] ?? false;
    $display_actions = func_get_args()[6] ?? false;
```